.github/workflows/test-action.yml

```
@@ -2,201 +1,200 @@
-  #   runs-on: ubuntu-latest
+  unit-test-action:
+    runs-on: ubuntu-latest
 
-  #   steps:
-  #     # Check out the repository code
-  #     - name: Checkout Code
-  #       uses: actions/checkout@v3
+    steps:
+      # Check out the repository code
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)